jdaviz/configs/imviz/plugins/links_control/links_control.vue

```diff
@@ -30,15 +41,88 @@
             persistent-hint>
           </v-switch>
         </v-row>
+        <div v-if="link_type_selected == 'WCS'">
+
+          <j-plugin-section-header>Select orientation</j-plugin-section-header>
+          <plugin-viewer-select
+            :items="viewer_items"
+            :selected.sync="viewer_selected"
+            :multiselect="multiselect"

```

Comment:
right now multiselect isn't available, except from the API.  Do we want to support it here or no?  If not, you can just pass `false`, remove the `multiselect` traitlet and remove the ternary in the `hint`.